src/branding-overrides/index.ts

```diff
@@ -15,10 +15,22 @@
  */
 import { injectGlobal } from 'react-emotion';
 
-export const applyWidgetBranding = () => {
+export const applyWidgetBranding = (backgroundColor: string, color: string) => {
   return injectGlobal`
     .Twilio .Twilio-MainContainer {
         width: 350px;
       }
+    .Twilio .Twilio-EntryPoint {
```

Comment:
How are we handling falling back to the Twilio defaults when nothing is set via the attributes? Should we omit parts of these style overrides if certain override attributes aren't set? 

And I'm assuming we probably don't want to override these 3 styles at all if neither color or backgroundColor is set?

Reply:
@stephenhand - I looked into this. What happened was that once any of color or backgroundColor returns null or undefined, the CSS fell back to the default. So there was no extra implementation needed for that.

Reply:
Yes but that's far from obvious for anybody reading the code, it at least should be commented. Approving on the assumption you will add a comment to this